src/NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.cpp

```diff
@@ -21,11 +26,43 @@ Mesh<Dim> mortar_mesh(const Mesh<Dim>& face_mesh1,
           Spectral::Quadrature::GaussLobatto};
```

Comment:
This function assumes that both meshes are on LGL points, correct? I suppose that's fine for now, but it would be good to add a comment about that. Also for the h-refined case where the face that covers the full mortar has less points than the larger face that is only half covered by the mortar, I think we talked about using Gauss points since both sides need to do an interpolation anyway.

BTW don't you have to adjust this function for h-refinement anyway, e.g. add a factor of 1/2 into the `std::max` call above if a face is larger than the mortar, or am I getting something wrong?

Reply:
1. An `ASSERT` would be safer than a comment, if anything.

2. You are indeed getting something wrong. Just because the mesh is half the size doesn't mean you can represent polynomials 2 * degree larger. This code is correct.

Reply:
#869

Reply:
👍

How about using Gauss points for when both elements have to interpolate anyway, which is when the larger element has a higher resolution than the smaller element?
- Do we want to do that?
- Will this mess up anything in the current code?
- Should we change it in this PR or open an issue?

Reply:
> Do we want to do that?

We should certainly try it.

> Will this mess up anything in the current code?

I don't think it will, but I haven't tested.

> Should we change it in this PR or open an issue?

Open an issue.  We have a looming deadline and I don't want to spend time right now swapping code that is known to work out for something I'm not sure will work.

(Your conditions on when double interpolation happens are correct.)

Reply:
Yes, we need to move on. Filing an issue is the best way to get this changed in the future, but as @wthrowe said, we have a deadline and we can't spend time on anything not on the critical path.

Reply:
#871